admiral/envs/components/examples/simple_particle.py

```diff
@@ -2,14 +2,14 @@
 from matplotlib import pyplot as plt
 import numpy as np
 
-from admiral.envs.components.agent import VelocityAgent, PositionAgent
+from admiral.envs.components.agent import VelocityAgent, PositionAgent, MassAgent, SizeAgent
 from admiral.envs.components.state import VelocityState, ContinuousPositionState
-from admiral.envs.components.actor import AccelerationMovementActor
+from admiral.envs.components.actor import AccelerationMovementActor, ContinuousCollisionActor
 from admiral.envs.components.observer import VelocityObserver, PositionObserver
 from admiral.envs import AgentBasedSimulation
 from admiral.tools.matplotlib_utils import mscatter
 
-class ParticleAgent(VelocityAgent, PositionAgent): pass
+class ParticleAgent(VelocityAgent, PositionAgent, MassAgent, SizeAgent): pass
```

Comment:
The movement of the agents is very uniform in this setting. Once I leave out the MassAgent the movement becomes random as expected. I have played with this and made a single agent that I just called PhysicsAgent with mass and size and it was random as well. Weird.